app-office/libreoffice/libreoffice-6.0.3.2.ebuild

```diff
@@ -316,6 +317,9 @@ src_unpack() {
 src_prepare() {
 	default
 
+	# Disable the LDAP backend extension used in Base
+	! use ldap && epatch "${FILESDIR}/${PN}-6.0.5.2-no-openldap.patch"
```

Comment:
this is an EAPI 6 ebuild, why are you using `epatch`?

Reply:
It now uses `eapply`.